skytap/vm_test.go

```diff
@@ -1116,6 +1375,46 @@ func createVMUpdateStructure() *UpdateVMRequest {
 	return opts
 }
 
+// createVMUpdateChangingVMNameAndRemovingThreeDisks creates an UpdateVMRequest will remove 2 disks specified on VMResponse.json
+// and update the VM name (without modifying any disks)
+func createVMUpdateChangingVMNameAndRemovingThreeDisks() *UpdateVMRequest {
```

Comment:
Not sure about this comment and the function name?  It talks about removing three disks and then the comment talks about two disks.

Reply:
True, it's because the method that updates the VM has two distinct steps:
1. changing VM name and adding/resizing disks 
2. removing disks

So this `UpdateVMRequest` will one change the name on the first step, and then it will remove
disks on the second.

Let me see if I can express this better.